crates/networking/rpc/engine/fork_choice.rs

```diff
@@ -130,8 +130,8 @@ fn parse(
     let forkchoice_state: ForkChoiceState = serde_json::from_value(params[0].clone())?;
     // if there is an error when parsing, set to None
     let payload_attributes: Option<PayloadAttributesV3> =
-        match serde_json::from_value::<PayloadAttributesV3>(params[1].clone()) {
-            Ok(attributes) => Some(attributes),
+        match serde_json::from_value::<Option<PayloadAttributesV3>>(params[1].clone()) {
+            Ok(attributes) => attributes,
```

Comment:
if you are having an Option, should you now match to `Some`?

Reply:
We have three scenarios:
A) We receive a PayloadAttributes ` Ok(Some(attributes))` -> We return Some(attributes)
B) We receive a null payload attributes `Ok(None)` -> We return None
C) We receive an invalid PayloadAttributes `Err()` -> We show an error message and return None
In both  A & B  we just return whatever is inside the Ok()